packages/backend/src/config/app.ts

```diff
@@ -26,6 +28,12 @@ export const Config = {
   coilApiGrpcUrl: envString('COIL_API_GRPC_URL', 'localhost:6000'),
   nonceRedisKey: envString('NONCE_REDIS_KEY', 'nonceToProject'),
 
+  accountsUrl: envString('ACCOUNTS_URL', 'http://127.0.0.1:3456'), // TODO default port?
```

Comment:
Need a ticket for this TODO?  Or do we track TODOs differently with a lint-like script or something?